api/src/pcapi/routes/backoffice_v3/templates/components/forms/checkbox_field.html

```diff
@@ -1,4 +1,4 @@
-<div class="form-check"
+<div class="form-check pc-{{ field.name }}"
```

Comment:
Suggestion : 
Est-ce qu'un préfixe plus clair ne permettrait pas d'éviter les éventuels conflits de noms ?
```suggestion
<div class="form-check pc-field-{{ field.name }}"
```